client/src/components/NavigationBar/Navigation.tsx

```diff
@@ -10,12 +10,11 @@ import LogoutOutlinedIcon from '@mui/icons-material/LogoutOutlined';
 //File will execute automatically
 import '../Styles/NavigationBar.scss';
 
-
 export const NavigationBar = () => {
     let tokenMap: Map<number, boolean> = new Map();
 
     //I use any here because it throws an error when trying to set the type to Map<number, boolean>
-    const [mapState, setMapState]: any = React.useState(null);
+    const [mapState, setMapState]: any = useState(null);
```

Comment:
is not a good practice to use type any in ts, when you hover over the element in this case setMapState it shows you the type